Jellyfin.Server/Migrations/Routines/MoveExtractedFiles.cs

```diff
@@ -76,27 +75,21 @@ public void Perform()
         // Make sure directories exist
         Directory.CreateDirectory(SubtitleCachePath);
         Directory.CreateDirectory(AttachmentCachePath);
-        do
+
+        await foreach (var result in context.BaseItems
```

Comment:
This query removes the limited select which was a significant performance gain for the migration

Reply:
readded